src/main/java/core/basesyntax/Application.java

```diff
@@ -2,6 +2,11 @@
 
 public class Application {
     public static void main(String[] args) {
-        // create three balls using class Lottery and print information about them in console
+        Lottery ball1 = new Lottery();
```

Comment:
You don't need to create three Lottery instances as only one can create as many balls as you want. Additionally, ball1 isn't a good name for this variable as it isn't an instance of the Ball class, it's an instance of the Lottery, so it should be named in an appropriate way (just a "lottery", for example).